ui/src/views/infra/AddPrimaryStorage.vue

```diff
@@ -870,6 +864,16 @@ export default {
           params['details[0].api_password'] = values.flashArrayPassword
           url = values.flashArrayURL
         }
+
+        if (values.provider === 'Linstor' || params.provider === 'Linstor') {
```

Comment:
don't we by now know that params.provider has been filled?

i.e.:
```suggestion
        if (params.provider === 'Linstor') {
```
?